test/loader/test_hgt_loader.py

```diff
@@ -175,13 +175,3 @@ def forward(self, x, edge_index, edge_weight):
     out2 = hetero_model(hetero_batch.x_dict, hetero_batch.edge_index_dict,
                         hetero_batch.edge_weight_dict)['paper'][:batch_size]
     assert torch.allclose(out1, out2, atol=1e-6)
-
-
-@withPackage('torch_sparse>=0.6.15')
```

Comment:
Couldn't see what this was testing that wasn't tested by adding the above line, so have removed it

Reply:
It tests that `HGTLoader` always returns the full set of node and edge types. We can disable this test for via `pytest.skipif` until the dataset is fixed.

Reply:
Okay. I've also added the same test in the above
```assert set(batch.edge_types) == set(data.edge_types)```
but makes sense to keep it

Reply:
I added just a skip - it didn't seem worth giving a condition. I'll instead open an issue now and note  this as a todo? WDTY?

Reply:
Yeah, that‘s what I meant. Thanks!

Reply:
Done here https://github.com/pyg-team/pytorch_geometric/issues/5251